workflows/energyplus.py

```diff
@@ -304,19 +307,21 @@ def run_energyplus(self, isIP, run_directory, file_name, args):
                     if os.path.exists(readvars_audit_path):
                         os.replace(readvars_audit_path, rv_audit_path)
 
-                # clean up
-                if os.path.exists(temp_rvi_path):
-                    os.remove(temp_rvi_path)
-                if os.path.exists(temp_mvi_path):
-                    os.remove(temp_mvi_path)
+                    # clean up things inside this IF block
```

Comment:
Similarly, all these variables were only declared inside `if os.path.exists(readvarseso_binary)` but they were attempting to be deleted from outside the block, so being used uninitialized.